src/tooltalk/apis/utils.py

```diff
@@ -49,7 +49,7 @@ def semantic_str_compare(prediction_text: str, ground_truth_text: str) -> bool:
         # initialize vectorizer only when needed
         _vectorize_text = _TextVectorizer()
 
-    prediction_vec = _vectorize_text(prediction_text)
+    prediction_vec = _vectorize_text(prediction_text or "")
```

Comment:
The evaluation dataset should be designed so that parameters that have strings in the ground truth should expect strings in the prediction. So desired behavior should be to fail predictions that do not predict strings for these instances. While passing an empty string is probably good enough, it seems safer to simply return `0` when `None` is predicted/

Reply:
Good point. Will update 👍